packages/webcert/src/components/error/modals/errorUtils.ts

```diff
@@ -22,7 +22,7 @@ export const uuidv4 = (): string => {
   })
 }
 
-export const messageSubstring = (activeError: ErrorData): string | undefined => {
+export const messageSubstring = (activeError: { message?: string }): string | undefined => {
```

Comment:
This won't introduce a regression bug. Instead of only allowing for the specific `ErrorData` we allow for a more generic object containing message that will still function with the `ErrorData` type :)

Reply:
Ok, nice :)